tests/sles4sap/trento/setup_jumphost.pm

```diff
@@ -78,8 +48,8 @@ sub run {
     assert_script_run("git clone $gitlab_clone_cmd . | tee " . GITLAB_CLONE_LOG);
 
     # Cypress.io installation
-    my $cypress_ver = get_var('TRENTO_CYPRESS_VERSION', '3.4.0');
-    cypress_install_container($cypress_ver);
+    my $cypress_ver = get_var('TRENTO_CYPRESS_VERSION', '4.4.0');
```

Comment:
That's a general suggestion:
```suggestion
    my $cypress_ver = get_var(TRENTO_CYPRESS_VERSION => '4.4.0');
```
If you use the "fat comma" `=>`, then you don't need to quote the string before.